src/app/model.ts

```diff
@@ -6,6 +6,10 @@ export interface BaseEntity {
     kind ? : string;
 }
 
+export interface Taggable {
```

Comment:
This file is generated, these changes will likely get lost.  

Generator code is here -> https://github.com/syndesisio/syndesis-ui/tree/master/scripts

another option is to maybe define this interface elsewhere, maybe we need a `model-extensions.ts` file where we can further enrich types as needed, then we'd have places to put  stuff like -> https://github.com/syndesisio/syndesis-ui/blob/master/src/app/store/step/step.store.ts#L4-L14 etc.